tests/src/main/java/com/linkedin/kmf/tests/BasicEndToEndTest.java

```diff
@@ -82,122 +83,122 @@ public void awaitShutdown() {
 
   /** Get the command-line argument parser. */
   private static ArgumentParser argParser() {
-    ArgumentParser parser = ArgumentParsers
-      .newArgumentParser("")
-      .defaultHelp(true)
-      .description("");
-
-    parser.addArgument("--topic")
-      .action(store())
-      .required(false)
-      .type(String.class)
-      .metavar("TOPIC")
-      .dest("topic")
-      .help("Produce messages to this topic and consume message from this topic");
-
-    parser.addArgument("--producer-id")
-      .action(store())
-      .required(false)
-      .type(String.class)
-      .dest("producerId")
-      .help("The producerId will be used by producer client and encoded in the messages to the topic");
+    ArgumentParser parser = ArgumentParsers.newArgumentParser("").defaultHelp(true).description("");
+
+    parser.addArgument("--topic").action(store()).required(false).type(String.class).metavar("TOPIC").dest("topic").help("Produce messages to this topic and consume message from this topic");
+
+    parser.addArgument("--producer-id").action(store()).required(false).type(String.class).dest("producerId").help("The producerId will be used by producer client and encoded in the messages to the topic");
 
     parser.addArgument("--broker-list")
-      .action(store())
-      .required(true)
-      .type(String.class)
-      .metavar("HOST1:PORT1[,HOST2:PORT2[...]]")
-      .dest("brokerList")
-      .help("Comma-separated list of Kafka brokers in the form HOST1:PORT1,HOST2:PORT2,...");
+        .action(store())
```

Comment:
It is probably because the use of tab. Have you changed configured idea to use only tab for indentation?

It is probably better to keep the 2-space indentation if possible. This makes code style consistent and avoid large chunk of code change in the pull request.